### PR TITLE
Add helpful dwarf-fortress caveats.

### PR DESCRIPTION
**Pull request by timvisher**
@jacknagel Here's the caveats as a separate pr.

### CODE REVIEW

File: dwarf-fortress.rb [hunk 1]
```diff
@@ -12,4 +12,10 @@ def install
     rm_rf "sdl" # only contains a readme
     libexec.install Dir["*"]
   end
+
+  def caveats; <<-EOS.undent
+    Requires an X11 environmment. Consider installing XQuartz via `brew cask install xquartz`
```
Comment (mistydemeo) [inline dwarf-fortress.rb]: I've added a specific X11 dependency - mind dropping this line?

**Comment by timvisher** [comment]
@mistydemeo no problem :)

What does :x11 get you?

**Comment by mistydemeo** [comment]
It explicitly checks for the availability of an X11 installation, and fails the dwarf-fortress installation if it's not present; it also provides a message instructing the user how to instal it.

**Comment by timvisher** [comment]
Neat :)

**Pull request closed without merging**